tests/testthat/test-convGDX2mif.R

```diff
@@ -1,5 +1,5 @@
 ## uncomment to skip test
-## skip("Skip GDX test")
+skip("Skip GDX test")
```

Comment:
why are you skipping the tests?

Reply:
tests are failing due to this line in reportTax:

```
# old tax reporting was deleted, stop if old tax reporting would be needed
  stopifnot(is.null(readGDX(gdx, name = "pm_tau_fe_tax_bit_st", format = "first_found", react = "silent")))

```

Reply:
I would remove the stop line instead of disabling the tests.

Every new remind run, from at least the last two months, replaced pm_tau_fe_tax_bit_st by pm_tau_fe_tax, pm_tau_fe_sub.
Even so, any gdx file stores not only the current run results but also any parameter that was previously saved to it, so this line will surely give false positives.

Assuming that the gdx used in the GDX test is a recent one, at a quick look I would not see a problem on running directly the updated tax reporting.

Reply:
The condition in `stopifnot` is exactly what decided between the old and new reporting. Since I removed the old reporting, as Renato suggested, `reportTax would just do nothing if the condition is FALSE, see on master branch: https://github.com/pik-piam/remind2/blob/9fa5b53869a6738af8fb6acae11e5af1ef741dcc/R/reportTax.R#L26
(the old reporting was in the `else` block)

Reply:
The test gdx that is automatically downloaded would test the old reporting, so we should probably update the gdx.